executor/include/executor/trait/is_executor.hpp

```diff
@@ -15,34 +15,48 @@
 
 namespace executor {
 
-template <class Executor, typename = void>
-struct is_executor : std::false_type {};
-
 namespace detail {
 const auto noop = [] {};
```

Comment:
static constexpr?

Reply:
Made it static const. constexpr not possible with lambda pre C++17 due to it being a non-literal type.

Reply:
Put a note on it :smile: